go.mod

```diff
@@ -134,12 +134,12 @@ require (
 	golang.org/x/crypto v0.7.0 // replaced
 	golang.org/x/exp v0.0.0-20230213192124-5e25df0256eb
 	golang.org/x/mod v0.9.0
-	golang.org/x/net v0.8.0
-	golang.org/x/oauth2 v0.6.0
+	golang.org/x/net v0.9.0
```

Comment:
(Note that this bumps a bunch of other /x/ packages that don't have release notes present here.)

Reply:
i have checked them and we should be fine